.github/ISSUE_TEMPLATE/0-bug-report.yaml

```diff
@@ -0,0 +1,311 @@
+        No need for backticks--this text will get formatted as a code-block.
+
+        What's a [dashboard panel](https://grafana.com/docs/grafana/latest/panels-visualizations/#panels-and-visualizations)?
+      placeholder: Copy "get-help" data here
+      value: Copy the panel's ["get-help" data](https://grafana.com/docs/grafana/latest/troubleshooting/send-panel-to-grafana-support/) here  
+  - type: markdown
+    attributes:
+      value: |
+        #
+        #
+        # Optional Questions:
+        #
+  - type: markdown
+    attributes:
+      value: |
+       #
+       # Did you check your grafana server logs?
+  - type: textarea
+    id: logs
+    attributes:
+      label: |
+        
+        ## Optional Questions:
+
+        ### Any relevant logs?
+
+      description: |
+        Please copy and paste any relevant log output. You might want to increase your Grafana log's verbosity to 'Debug'
+        No need for backticks--this text will get formatted as a code-block.
+        [How do I find my grafana server logs](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#logs)?
+        [How do I increase my grafana server logs to 'debug'](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#log)?
+      render: sh
+      placeholder: Copy your grafana server logs here
+      value: Copy your grafana server logs here  
+  - type: markdown
+    attributes:
+      value: |
+        
+        # Can you share more configuration details?
+        
+  - type: dropdown
+    id: gf-deployment
+    attributes:
+      label: Grafana Platform?
+      description: |
+        **How** are you running/deploying Grafana?
+      options:
+        - I use Grafana Cloud
+        - Docker
+        - Kubernetes
+        - A package manager (APT, YUM, BREW, etc.)
+        - A downloaded binary
+        - Other
+        - I don't know
+    validations:
+      required: false
+  - type: dropdown
+    id: user-os
+    attributes:
+      label: User's OS?
+      description: What operating system are you running locally?
+      options:
+        - MacOS
+        - Windows
+        - Ubuntu
+        - Debian
+        - RHEL
+        - CentOS Stream
+        - Fedora
+        - Manjaro
+        - OpenSUSE
+        - Rocky
+        - Linux Mint
+        - Arch Linux
+        - Raspberry Pi OS
+        - Other Linux distro
+        - Other (non-Linux)
+        - I don't know
+  - type: dropdown
+    id: user-browser
+    attributes:
+      label: User's Browser?
+      description: Is the bug occuring in Grafana's user-interface? If so, what browsers are you seeing the problem on? You may choose more than one.
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
```

Comment:
Nit: No other browsers have been stylized with the corporate name, if we want to be consistent about this, we should probably also expand Google Chrome and Mozilla Firefox.